build/cmake/CondorPackageConfig.cmake

```diff
@@ -490,7 +490,7 @@ elseif( "${OS_NAME}" STREQUAL "LINUX" AND CONDOR_PACKAGE_BUILD )
 		set( C_BIN			usr/bin )
 		set( C_LIBEXEC		usr/libexec/condor )
 		set( C_LIBEXEC_BLAHP		usr/libexec/blahp )
-		if ( ${LINUX_NAME} MATCHES "Fedora"  AND  ${LINUX_VER} GREATER 40 )
+		if ( ${LINUX_NAME} MATCHES "Fedora"  AND  ${LINUX_VER} GREATER 41 )
```

Comment:
Instead of switching on the distro -- who knows which other distros are going to get rid of `/usr/sbin` ??, how about we do something like

```
if (IS_SYMLINK "/usr/sbin")
set (C_SBIN usr/bin)
else()
set (C_SBIN usr/sbin
endif()
```

Reply:
Love it!